.size-limit

```diff
@@ -0,0 +1,24 @@
+[
```

Comment:
Please also add workflow for it

https://github.com/reactchartjs/react-chartjs-2/blob/master/.github/workflows/checks.yml#L7-L25

Reply:
@thabarbados after merge into master all will ok